static/js/components/inputs/DateField.js

```diff
@@ -147,57 +147,47 @@ export default class DateField extends React.Component {
     }
 
     const edit = getEditObject()
+    const error = _.get(errors, keySet, undefined) !== undefined
```

Comment:
Is there no option for errorText anymore?

Reply:
I didn't find an equivalent for errorText.